vector/src/main/res/values/strings.xml

```diff
@@ -952,6 +952,9 @@
     <string name="room_one_user_is_typing">%s is typing…</string>
     <string name="room_two_users_are_typing">%1$s &#038; %2$s are typing…</string>
     <string name="room_many_users_are_typing">%1$s &#038; %2$s &#038; others are typing…</string>
+    <!--TODO #3296 add next two strings values -->
```

Comment:
This TODO is not necessary. Adding new strings will be handled automatically by Weblate

Reply:
Done. 
Thanks !